frontend/hooks/useServices.ts

```diff
@@ -89,12 +94,24 @@ export const useServices = () => {
 
   const deleteServiceState = (serviceHash: ServiceHash) =>
     setServices((prev) => prev?.filter((s) => s.hash !== serviceHash));
+  const service = services?.[0];
+
+  const isServiceNotRunning = useMemo(
```

Comment:
wouldn't it be easy just to go with `isServiceRunning` instead? b/c the way it's used is a bit harder to read especially having this comment

```
// if agent is running, no need to show notification
if (!isServiceNotRunning) return;